vimeo/vimeo.py

```diff
@@ -53,6 +41,39 @@ def resource_string(self, path):
         data = pkg_resources.resource_string(__name__, path)
         return data.decode("utf8")
 
+    def create_fragment(self, html, context={}, js=[], css=[], initialize=None):
+        """
+        Create an XBlock Fragment
+
+        Creates an XBlock Fragment given an HTML file path, 
+        an optional Context, optional JS file paths, optional CSS file paths,
+        and an optional Initialize JS value
+
+        Arguments:
+            html (str): HTML File Path
+            context (dict): Optional Context for HTML File
+            js (list): Optional List of JS File Paths
+            css (list): Optional List of CSS File Paths
+            initialize (str): Optional Initialize Value for JS
+
+        Returns:
+            Fragment 
+        """
+        html = Template(self.resource_string(html))
```

Comment:
@nizarmah If python3 is not used, the html content will not be encoded with utf-8 by default. This could potentially break if the html content has unicode characters. Might make sense to add a check for the python version used and call `load_resource` if the version in below 3.